tests/integration/test_log_query_probability/test.py

```diff
@@ -45,7 +55,9 @@ def test_log_quries_probability_one(start_cluster):
     node1.query("TRUNCATE TABLE system.query_log")
 
 
-def test_log_quries_probability_two(start_cluster):
+def test_log_queries_probability_two(start_cluster):
+    ensure_query_log_exists()
```

Comment:
I don't understand why this is necessary. There is a `node2.query("SYSTEM FLUSH LOGS")` below, which will create the table

Reply:
Yes, but as far as I understood from logs that query returns before the table is actually created.

Reply:
AFAIK `SYSTEM FLUSH LOGS` is supposed to guarantee that anything logged before is immediately available. No matter if the table needs to be created or not